crates/llama-cpp-bindings/build.rs

```diff
@@ -69,7 +69,14 @@ fn main() {
     }
 
     let dst = config.build();
-    println!("cargo:rustc-link-search=native={}/build", dst.display());
+    if cfg!(target_os = "windows") {
+        #[cfg(debug_assertions)]
+        println!("cargo:rustc-link-search=native={}\\build\\Debug", dst.display());
+        #[cfg(not(debug_assertions))]
+        println!("cargo:rustc-link-search=native={}\\build\\Release", dst.display());
```

Comment:
Turns out, the folder for `llama.lib` & `ggml_static.lib` are different depending which build mode is using on windows platform, compared to linux platform, the libs are always generated under `tabby/target/[debug or release]/build/llama-cpp-bindings-7425e10c6b84e425/out/build`.